docs/faqs.md

```diff
@@ -385,61 +357,6 @@ Keep reading in the [consuming recipes section](consuming_recipes.md#isolate-you
 Version ranges are currently allowed on a handful of dependencies, but not for general use.
 See [Dependencies Version Ranges](adding_packages/dependencies.md#version-ranges) for additional details.
 
-## How to consume a graph of shared libraries?
```

Comment:
It's the default in CCI

Reply:
mmm but possibly still needs pointing to `-o "*:shared=True"` as a blanket recommendation?

Reply:
Addressed by 8722871da21b465c8c5be849e5c1fb322b755ceb